packages/plugin-hardhat/src/admin.ts

```diff
@@ -36,8 +36,12 @@ export function makeChangeProxyAdmin(hre: HardhatRuntimeEnvironment, platformMod
     const proxyAdminAddress = await getAdminAddress(hre.network.provider, proxyAddress);
 
     if (manifestAdminAddress !== proxyAdminAddress) {
-      throw new Error('Proxy admin is not the one registered in the network manifest');
-    } else if (manifestAdminAddress !== newAdmin) {
+      console.log(
+        `Proxy admin ${proxyAdminAddress} is not the one registered in the network manifest ${manifestAdminAddress}`,
+      );
+    }
+
+    if (proxyAdminAddress !== newAdmin) {
       const overrides = opts.txOverrides ? [opts.txOverrides] : [];
       await admin.changeProxyAdmin(proxyAddress, newAdmin, ...overrides);
```

Comment:
This should no longer use the admin from the manifest, but should use the proxy's current admin.  For example, you can remove lines 34-35, then change this to:
```suggestion
      const AdminFactory = await getProxyAdminFactory(hre, signer);
      const admin = attach(AdminFactory, proxyAdminAddress); 
      await admin.changeProxyAdmin(proxyAddress, newAdmin, ...overrides);
```